tests/Foundation/FoundationMakesHttpRequestsJsonTest.php

```diff
@@ -81,6 +81,20 @@ public function testSeeJsonEqualsWithNonJson()
 
         $this->fail('Invalid JSON should fail seeJsonEquals');
     }
+
+    public function testSeeJsonResponseOnString()
+    {
+        $this->response = new Illuminate\Http\Response(['foo']);
+
+        try {
+            $this->dontSeeJson(['foo']);
+            $this->seeJsonContains(['foo']);
+        } catch (PHPUnit_Framework_AssertionFailedError $e) {
+            $this->assertStringStartsWith('Invalid JSON', $e->getMessage());
```

Comment:
Do we necessarily want to be testing exception messages?

I thought this was generally bad practice/frowned upon.